README.md

```diff
@@ -94,8 +99,8 @@ Please also refer to [kubectl official doc](https://kubernetes.io/docs/reference
 
 ## Supported commands
 
-Checked: Supported and works in current latest version  
-Unchecked: Will be supported but it's still under development  
+Checked: Supported and works in current latest version
+Unchecked: Will be supported but it's still under development
```

Comment:
Please remove these diff; they are intended to break the line.

Reply:
That is IDE automatically trimmed extra whitespace at the end of a line.
What about `\` at the end? It is more visible IMHO.
I have updated README

Reply:
Thank you, it's much better!